### PR TITLE
changes the shape of the button dark mode

### LINKED ISSUE #53: Style and move the dark mode button
We'd like to:

- Change the current toggle button a dark mode toggle button that looks like [this](https://www.npmjs.com/package/react-dark-mode-toggle) (can use this package or any other method you find as long as the result is a dark mode button that looks like that)
- Move the position of the button so it's next to the GitHub icon on the right

Currently it looks like this:
Screen Shot 2020-10-17 at 7 45 36 AM

### PR DESCRIPTION
**Pull request by jackson541**
close #53

### CODE REVIEW

**Comment by jackson541** [comment]
image

File: src/components/App.jsx [hunk 1]
```diff
@@ -150,16 +150,15 @@ class App extends React.Component {
...
+        darkMode: true,
       });
-    }
-    else{
-      document.body.className = "";
+    } else {
+      document.body.className = '';
       this.setState({
-        darkMode: false
+        darkMode: false,
```
Comment (trybick) [inline src/components/App.jsx:174]: Thanks for getting these formatting changes in 👍  I might need to add an issue to a pre-commit hook for linting

File: src/components/core/Header.jsx [hunk 2]
```diff
@@ -1,30 +1,41 @@
...
-    <h2 className="page-header-title">
-      <span className="page-header-title-highlight">Issue</span>-Collab
-    </h2>
-    <div>
-      <Toggle
-          onChange={onToggleChangeDarkMode}
-      /> 
-      <span className="toggle-label">dark mode</span>
+const Header = ({ onToggleChangeDarkMode, darkMode }) => {
+  const [isDarkMode, setIsDarkMode] = useState(() => false);
```
Comment (trybick) [inline src/components/core/Header.jsx]: Hmm, we are storing darkMode state in App and it's being passed to Header. Do we need another instance of isDarkMode here? Can we just use the darkMode prop?

**Review by trybick — COMMENTED** [review]
I have a couple comments but this looks really good! 

Also, could we try decreasing the size of the new toggle? It looks really good but it might be a little big. I think it has a `size` prop that can be passed to that component.

**Comment by trybick** [comment]
Also sorry there are some merge conflicts, this is the PR that created them if it helps to look at it - https://github.com/trybick/issue-collab/pull/56

**Comment by jackson541** [comment]
@trybick Of course, I can change your size to be more proportional. I will also resolve conflicts, thank you!

File: src/components/core/Header.jsx (hunk 2 above)
Comment (jackson541) [reply src/components/core/Header.jsx]: It really works! I ended up letting it go unnoticed.

**Comment by trybick** [comment]
Awesome @jackson541 thanks!

**Pull request merged**